dist2src/worker/celer.py

```diff
@@ -0,0 +1,33 @@
+# Copyright Contributors to the Packit project.
+# SPDX-License-Identifier: MIT
+
+from os import getenv
+
+from celery import Celery
+from dist2src.worker.sentry import configure_sentry
+from lazy_object_proxy import Proxy
+
+
+class Celer:
```

Comment:
This is named `Celer` on purpose?

Reply:
yes, there's already `celery` module with `Celery` class, so we need a different name for the file and class. Do you like `Celerizer` (as in p-s) more?

Reply:
>  Do you like Celerizer (as in p-s) more?

it sounds better to me, but that's just a nitpick, I am ok with `Celer` as well